build.assets/Dockerfile-fips

```
@@ -1,6 +1,61 @@
 # This Dockerfile makes the FIPS "build box": the container used to build official
 # FIPS releases of Teleport and its documentation.
 
+
+FROM ubuntu:18.04 as boringssl
+# The below tools are required in order to build and compile the module:
+# Clang compiler version 7.0.1
+# Go programming language version 1.12.7
+# Ninja build system version 1.9.0
+#
+# We also need the FIPS 140-2 validated release of BoringSSL: ae223d6138807a13006342edfeef32e813246b39
+# For more information please refer to the section 12. Guidance and Secure Operation of:
+# https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp3678.pdf
+
+RUN apt-get update -y --fix-missing && \
+    apt-get -q -y upgrade && \
+    apt-get install -y --no-install-recommends apt-utils ca-certificates curl && \
+    apt-get install -q -y --no-install-recommends \
+        build-essential \
+        cmake \
+        git \
+        tar \
+        xz-utils \
+        unzip \
+        zip \
+        && \
+    apt-get -y clean && \
+    rm -rf /var/lib/apt/lists/*
+
+
+ARG LLVM_VERSION=7.0.1
+RUN mkdir -p /opt && cd /opt && curl -L https://releases.llvm.org/7.0.1/clang+llvm-7.0.1-x86_64-linux-gnu-ubuntu-18.04.tar.xz | tar xJ 
+ENV PATH="$PATH:/opt/clang+llvm-7.0.1-x86_64-linux-gnu-ubuntu-18.04/bin"
```

You want this path to be at the beginning, not the end:
```suggestion
ENV PATH="/opt/clang+llvm-7.0.1-x86_64-linux-gnu-ubuntu-18.04/bin:$PATH"
```